services/frontend/src/scenes/ProjectDetails/ProjectDetails.scss

```diff
@@ -11,43 +11,83 @@
     display: flex;
     flex: 1;
     flex-direction: column;
-    margin: 0 $default-spacing*2;
+    margin: 0 $default-spacing*4;
+    padding-top: $default-spacing;
+
 
     .project-details-content-title {
         color: $color-neutral-6;
     }
 
     .project-details-card:first-child {
-        padding-top: $default-spacing;
+        margin-top: $default-spacing;
     }
+}
 
-    .project-details-card {
+.project-details-card {
+    display: flex;
+    flex: 1;
+    margin: $default-spacing*2 0;
+    padding: $default-spacing;
+}
+
+#project-info {
+    display: flex;
+    //justify-content: center;
+
+    .project-info-card-content {
         display: flex;
-        padding: $default-spacing*2;
-        margin: $default-spacing*2 0;
-        flex-direction: column;
 
-        .project-info-card-content {
+        .project-card-info-active-section {
+            align-self: flex-start;
             display: flex;
+            flex: 1;
+        }
 
-            .project-card-info-active-section {
-                align-self: flex-start;
-                display: flex;
-                flex: 1;
-            }
-
-            .project-info-card-content-section {
-                flex: 3;
-                padding-left: $default-spacing*2;
+        .project-info-card-content-section {
+            flex: 3;
+            padding-left: $default-spacing*2;
 
-                .project-info-card-name {
-                    margin-bottom: $default-spacing;
-                }
+            .project-info-card-name {
+                margin-bottom: $default-spacing;
+            }
 
-                .project-info-card-description {
-                    color: $color-text-secondary;
-                }
+            .project-info-card-description {
+                color: $color-text-secondary;
             }
         }
     }
-}
\ No newline at end of file
+}
+
+#project-contributors{
```

Comment:
All these sectional classes could be a lot cleaner by combining them together under one class, and then using additional classes for more specific styling; right now, it's a lot of duplication.